server/util/redact/redact.go

```diff
@@ -213,6 +224,16 @@ func stripRepoURLCredentialsFromBuildMetadata(metadata *bespb.BuildMetadata) {
 			metadata.Metadata[repoURLKey] = gitutil.StripRepoURLCredentials(val)
 		}
 	}
+	if m, ok := metadata.Metadata[explicitCommandLineName]; ok {
+		var commandLine []string
+		json.Unmarshal([]byte(m), &commandLine)
```

Comment:
(this can return an error - it's safe to ignore but worth clarifying it with `_`)

```suggestion
		_ = json.Unmarshal([]byte(m), &commandLine)
```